tests/phpunit/tests/date/dateI18n.php

```diff
@@ -7,11 +7,22 @@
  */
 class Tests_Date_DateI18n extends WP_UnitTestCase {
 
+	/**
+	 * Clean up.
+	 */
+	public function tear_down() {
+		// Reset changed options to their default value.
+		update_option( 'gmt_offset', 0 );
+		update_option( 'timezone_string', '' );
```

Comment:
Oh, missed this one. Same question as earlier regarding `date_default_timezone_set( 'UTC' );`.

Reply:
same answer